Change `OMPL_INCLUDE_DIR` to `OMPL_INCLUDE_DIRS`

`mod` is currently failing to build for Humble on the ROS 2 buildfarm: https://build.ros2.org/view/Hbin_uJ64/job/Hbin_uJ64__mod__ubuntu_jammy_amd64__binary/39/console

```
23:32:09 In file included from /tmp/binarydeb/ros-humble-mod-1.1.0/include/ompl/mod/objectives/IntensityMapOptimizationObjective.h:21,
23:32:09                  from /tmp/binarydeb/ros-humble-mod-1.1.0/src/ompl/mod/objectives/IntensityMapOptimizationObjective.cpp:19:
23:32:09 /tmp/binarydeb/ros-humble-mod-1.1.0/include/ompl/mod/objectives/MoDOptimizationObjective.h:21:10: fatal error: ompl/base/OptimizationObjective.h: No such file or directory
23:32:09    21 | #include <ompl/base/OptimizationObjective.h>
23:32:09       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
23:32:09 compilation terminated.
```

It uses `ompl` 1.7.0. The last time `mod` was successfully built on the buildfarm, it used `ompl` 1.6.0: https://build.ros2.org/view/Hbin_uJ64/job/Hbin_uJ64__mod__ubuntu_jammy_amd64__binary/13/. I looked at the differences between `1.6.0` and `1.7.0`: https://github.com/ompl/ompl/compare/1.6.0...1.7.0. They removed a lot of legacy CMake variables and kept some to avoid breaking downstream packages (https://github.com/ompl/ompl/pull/1246), but ultimately they did remove `OMPL_INCLUDE_DIR`: https://github.com/ompl/ompl/compare/1.6.0...1.7.0#diff-e9cf754273efbc3a522dd1900e59b8c38ae5158cc00a93c867cd7304930bb1be. `OMPL_INCLUDE_DIR` is used by `mod` here: https://github.com/OrebroUniversity/mod/blob/191e37aca08e7e8cb33121c868513debc490e12b/CMakeLists.txt#L26.

The solution is to change `OMPL_INCLUDE_DIR` to `OMPL_INCLUDE_DIRS`. I've tested in an Ubuntu 22.04 container with Humble and `ompl` 1.7.0 (built from source since you can only get 1.6.0 from the main/non-testing ROS 2 apt repo currently), and it fixes the issue. I saw that your [repo is actually on Bitbucket](https://bitbucket.org/mapsofdynamics/mod/src/master/) and that this is just a mirror, so I just opened an issue.

After this is fixed, a new (Humble) release would be appreciated. I plan to do a Humble sync this week, so your package will be removed from the apt repo if it isn't fixed and released by then.

@ksatyaki FYI

--- a/humble/distribution.yaml
+++ b/humble/distribution.yaml
@@ -5192,7 +5192,7 @@ repositories:
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/OrebroUniversity/mod-release.git
-      version: 1.1.0-4
+      version: 1.1.1-1
     source:
       test_pull_requests: true
       type: git

--- a/humble/distribution.yaml
+++ b/humble/distribution.yaml
@@ -5192,7 +5192,7 @@ repositories:
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/OrebroUniversity/mod-release.git
-      version: 1.1.0-1
+      version: 1.1.0-4
     source:
       test_pull_requests: true
       type: git

